lib/pos_mempool.go

```diff
@@ -169,6 +174,11 @@ func (mp *PosMempool) Start() error {
 
 	// Create the transaction register, the ledger, and the nonce tracker,
 	mp.txnRegister = NewTransactionRegister(mp.globalParams)
+	mp.feeEstimator = NewPoSFeeEstimator()
+	// TODO: parameterize num blocks. Also, how to pass in blocks.
+	if err := mp.feeEstimator.Init(mp.txnRegister, []*MsgDeSoBlock{}, 1, mp.globalParams); err != nil {
```

Comment:
You can create a separate `PosMempool.Init(...)` function that can take in all of the initialization params for the mempool and the initialization params for the estimator. It can be separate from the `PosMempool.Start()` which can be scoped to just starting internal event loop for the mempool.

Suggestion:

```
func (mp *PosMempool) Init(
	params *DeSoParams, 
	globalParams *GlobalParamsEntry, 
	readOnlyLatestBlockView *UtxoView,
	latestBlockHeight uint64,
	dir string,
	inMemoryOnly bool,
	maxMempoolPosSizeBytes uint64,
	mempoolBackupIntervalMillis uint64, 
	blocks []*MsgDeSoBlock, // The last n blocks at the tip of the blockchain
)
```